spark/src/main/scala/ai/chronon/spark/streaming/JoinSourceRunner.scala

```diff
@@ -284,13 +287,23 @@ class JoinSourceRunner(groupByConf: api.GroupBy, conf: Map[String, String] = Map
     Some(sorted(k))
   }
 
+  def repartition(stream: DataStream): DataStream = {
+    if (microBatchRepartition <= 0) {
+      stream
+    } else {
+      logger.info(s"Repartitioning stream to $microBatchRepartition partitions")
```

Comment:
nit: maybe use debug log level?
```suggestion
      logger.debug(s"Repartitioning stream to $microBatchRepartition partitions")
```

Reply:
hmm well maybe info is still better. debug i feel is for extra granular, redundant information that we don't want to show in production.